pom.xml

```diff
@@ -22,13 +22,22 @@
     </repository>
   </repositories>
 
-  <artifactId>fcrepo-swordserver</artifactId>
+  <artifactId>fcrepo4-swordserver</artifactId>
   <name>Fedora Repository SWORD Provider</name>
   <description>
     Service implementing a SWORD endpoint for Fedora Commons Repositories.
   </description>
   <packaging>bundle</packaging>
 
+  <scm>
+    <connection>scm:git:git://github.com/fcrepo4-labs/${project.artifactId}.git
+    </connection>
+    <developerConnection>scm:git:git@github.com:fcrepo4-labs/${project.artifactId}.git
+    </developerConnection>
+    <url>https://github.com/fcrepo4-labs/fcrepo4-swordserver</url>
```

Comment:
Actually, I am not sure the SCM block gets inherited during the release process. It would be a shame to remove these blocks expecting them to be inherited, then at release time having to add them all back in.
These are all of the projects that could potentially use the `${project.artifactId}` in the `url` element.